code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -494,10 +494,6 @@ func (sc *StorageSmartContract) verifyChallenge(t *transaction.Transaction,
 			"can't get related allocation: %v", err)
 	}
 
-	if alloc.Finalized {
-		return "", common.NewError(errCode, "allocation is finalized")
```

Comment:
So we'll still process the challenge even the alloc is finalized?

Reply:
No, I added this change and it accidently removed when I was merging to staging.

Reply:
Readded it